migration/tests/tests.rs

```diff
@@ -0,0 +1,24 @@
+#![cfg(test)]
+
+use migration::{Migrator, MigratorTrait};
+use test_context::test_context;
+use test_log::test;
+use trustify_common::db::test::TrustifyContext;
+
+#[test_context(TrustifyContext, skip_teardown)]
+#[test(tokio::test)]
+async fn test_migrations(ctx: TrustifyContext) -> Result<(), anyhow::Error> {
+    let db = ctx.db;
+
+    let migrations = Migrator::get_applied_migrations(&db).await?;
+    // 'Migrator.up' was called in bootstrap function when using TrustifyContext.
+    // At this point we already have migrations.
+    assert!(migrations.len() > 1);
+
+    db.refresh().await?;
```

Comment:
I'm assuming you've removed a `down` function in one of our migrations locally and run this test just to make sure it fails? I'm assuming line 18 will panic, yes?

Reply:
ok about the expected behavior I would say -- yes.

```
       START             trustify-migration::tests test_migrations

running 1 test
Error: Migration Error: We Don't Do That Here
test test_migrations ... FAILED

failures:

failures:
    test_migrations

test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 1.58s

        FAIL [   1.585s] trustify-migration::tests test_migrations
   Canceling due to test failure
------------
     Summary [   1.585s] 1 test run: 0 passed, 1 failed, 74 skipped
        FAIL [   1.585s] trustify-migration::tests test_migrations
error: test run failed
```

But the missing `down` AFAIK is the problem and this test will break the CI build and etc.. so we can fix the migration.

Reply:
that is a bit weirdo test because probably we can call this as a cross-cutting-concern test instead of unit/integration/business-related test...

Reply:
> But the missing `down` AFAIK is the problem and this test will break the CI build and etc.. so we can fix the migration.

Yes, that's the exact behavior we want. CI should fail if we forgot a `down` migration.